docs/sources/administration/cli.md

```diff
@@ -104,39 +104,38 @@ grafana-cli --insecure --pluginUrl https://company.com/grafana/plugins/<plugin-i
 
 ### Enable debug logging
 
-`--debug` or `-d` enables debug logging.
-> Where does is the log stored? Why would you want to use this? Is my example in any way realistic?      
+`--debug` or `-d` enables debug logging. Debug output is returned and shown in the terminal.  
 
 **Example:**
 ```bash
 grafana-cli --debug plugins install <plugin-id>
 ```
 
-### --configOverrides value  
-> In plain terms, what does this mean?
+### Override a configuration setting  
```

Comment:
@daniellee , please take a look at this one.

Reply:
When starting Grafana, you have several layers when it comes to configuration. You can use a couple of command line arguments to the Grafana server binary, the config file contains most of the settings and lastly environmental variables can override config file settings.

`configOverrides` is a weird command line argument for the cli binary that acts like an environmental variable override. The only good use cases I can think of are to redirect logging to another file (maybe to log plugin installs in a service like Hosted Grafana) or when resetting the admin password and you have non-default values for some important config value (like where the database is located).

Reply:
Updated.